apps/src/gamelab/AnimationTab/AnimationTab.jsx

```diff
@@ -86,4 +91,10 @@ var AnimationTab = React.createClass({
     );
   }
 });
-module.exports = Radium(AnimationTab);
+module.exports = connect(state => ({
+  columnSizes: state.animationTab.columnSizes
+}), dispatch => ({
+  onColumnWidthsChange: (widths) => {
+    dispatch(setColumnSizes(widths));
+  }
```

Comment:
for state and dispatch, you didnt put parens around your single param, but for widths you did.